src/main/java/uk/nhs/tis/sync/job/PersonRecordStatusJob.java

```diff
@@ -36,32 +28,22 @@
 @Component
 @ManagedResource(objectName = "sync.mbean:name=PersonRecordStatusJob",
     description = "Job set a Person's (Training Record) Status if their programme membership(s) started/ended")
-public class PersonRecordStatusJob implements RunnableJob {
+public class PersonRecordStatusJob extends PersonCurrentPmSyncJobTemplate {
 
-  protected static final int DEFAULT_PAGE_SIZE = 5000;
   private static final Logger LOG = LoggerFactory.getLogger(PersonRecordStatusJob.class);
-  private static final int FIFTEEN_MIN = 15 * 60 * 1000;
-  private static final String BASE_QUERY =
-      "SELECT DISTINCT personId FROM ProgrammeMembership" + " WHERE personId > :lastPersonId"
-          + " AND (programmeEndDate = ':endDate' OR programmeStartDate = ':startDate')"
-          + " ORDER BY personId LIMIT :pageSize";
-  private static final String FULL_SYNC_DATE_STR = "ANY";
-  private static final String NO_DATE_OVERRIDE = "NONE";
 
   private final EntityManagerFactory entityManagerFactory;
   private final ObjectMapper objectMapper;
 
-  @Value("${application.jobs.personRecordStatusJob.dateOfChangeOverride:}")
+  @Value("${application.jobs.personRecordStatusJob.dateOfChangeOverride}")
```

Comment:
I don't know why you've removed the default.  It is "belt and braces" but is a useful way of establishing a default

Reply:
I thought the default has already been provided in` application.yml` and when I searched for other usage of `@Value`, I couldn't find any where else except for a default `pageSize 8000` in `personElasticSearchSyncJob`, so I removed the colon. If it's better to have it, I will change it back.